_data/changelogs/component-in-page-navigation.yml

```diff
@@ -2,6 +2,17 @@ title: In-page navigation
 type: component
 changelogURL:
 items:
+  - date: 2023-02-03
+    summary: Component updated to allow for any element to be used as the main content section.
+    summaryAdditional: The previous version of the component required the main content section to be a `<main>` element. This update allows for any element to be used as the main content section (e.g., `<div>`) by adding the class `.usa-in-page-nav__main` to it.
+    isBreaking: true
+    affectsGuidance: true
+    affectsJavascript: true
+    affectsMarkup: true
+    affectsStyles: true
+    githubRepo: uswds
+    githubPr: 5124
+    versionUswds: 3.3.0
```

Comment:
Note: Before merge, the version and date will need to be updated for this changelog item